### ISSUE TITLE
Modeule not found after upgrading to new version 3.0.0

### ISSUE THREAD

**Issue by alii-gator** [redhat-developer/vscode-extension-tester#115]
OS: `Mojave 10.14.6`
I upgraded from `2.6.0 -> 3.0.0`

Now when i try to execute tests i get the following error
```
code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/sample-projects/suitecloud-vs-code-extension-tests/node_modules/vscode-extension-tester/out/extester.js',
    '/sample-projects/suitecloud-vs-code-extension-tests/node_modules/vscode-extension-tester/out/cli.js'
  ]
```

**Comment by alii-gator** [comment]
I switched back to version `2.6.0` and it works fine.

**Comment by jrichter1** [comment]
There is not much to go on here, it could easily have been a problem with installation, looking at the error.
Could you try again with a fresh install and see if any errors creep up along the way?
 - remove `node_nodules` and `packag-lock.json`
 - upgrade the dependency in `package.json` to `3.0.0`
 - run `npm install`
 - run the tests

Thanks

**Comment by dcermak** [comment]
I've ran into the following issue when upgrading to `^3`:
``` ShellSession
node_modules/vscode-extension-tester/out/extester.d.ts:6:15 - error TS2307: Cannot find module 'vscode-extension-tester-native' or its corresponding type declarations.

6 export * from 'vscode-extension-tester-native';
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
That is trivially solved via `yarn add --dev vscode-extension-tester-native` but makes me wonder whether vscode-extension-tester shouldn't then depend on vscode-extension-tester-native?

**Comment by jrichter1** [comment]
Native was split off so people who don't want to use it don't have to build it (there is no prebuild module for node 13+ sadly).

https://github.com/redhat-developer/vscode-extension-tester#migrating-to-3x

**Comment by jrichter1** [comment]
should be fixed in 3.0.1
